addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3734,7 +3744,19 @@ public Action:TF2Items_OnGiveNamedItem(client, String:classname[], iItemDefiniti
 		}
 	}
 
-	if(!StrContains(classname, "tf_weapon_medigun") && (iItemDefinitionIndex!=35 || iItemDefinitionIndex!=411 || iItemDefinitionIndex!=998))  //Kritzkrieg, Quick Fix, Vaccinator
+	if(!StrContains(classname, "tf_weapon_syringegun_medic")) // Syringe Guns
+	{
+		new Handle:itemOverride=PrepareItemHandle(item, _, _, "17 ; 0.05 ; 144 ; 1");
+			//17: 5% uber on hit
+			//144: Sets weapon mode?????
+		if(itemOverride!=INVALID_HANDLE)
+		{
+			item=itemOverride;
+			return Plugin_Changed;
+		}	
+	}
+
+	if(!StrContains(classname, "tf_weapon_medigun"))  // Mediguns
```

Comment:
This can be an else-if now since there's no way a weapon is going to be named `tf_weapon_syringegun_medic_tf_weapon_medigun`.  At least there better not be -.-